bbb-install.sh

```diff
@@ -895,9 +895,10 @@ server {
     ssl_certificate_key /etc/letsencrypt/live/$HOST/privkey.pem;
     ssl_session_cache shared:SSL:10m;
     ssl_session_timeout 10m;
-    ssl_protocols TLSv1.2;
+    ssl_protocols TLSv1.2 TLSv1.3;
     ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
     ssl_prefer_server_ciphers on;
+    ssl_ecdh_curve secp521r1:secp384r1;
```

Comment:
We shouldn't be setting `ssl_ecdh_curve` here. The default is `auto`, which is fine - it uses OpenSSL's built-in list.

Note that explicitly listing some here might make some types of ECDSA certificates incompatible .

Reply:
We could remove this line, but the install script does not issue ECDSA certificates.
If ECDSA certs are issued after the script ran, the nginx config has to be modified nonetheless.

What about commenting out this line and leave a explanatory comment. What do you think?

Reply:
Again, I recommend just completely leaving the `ssl_ecdh_curve` setting out. The default "auto" setting with the openssl version use with nginx on ubuntu 18.04 is fine, and still gives you the same rank on the SSLLabs test (and the setting you've used has actually disabled some perfectly fine curves like x25519).

Reply:
Specifically, the curves disabled by this setting are `secp256r1`, `x25519`, and `x448`. If you're using a cert with a key size >2048 bits rsa you might want to consider disabling the 256-bit curves (`secp256r1` and `x25519`), but that's something that should be done by a specific person on their own server rather than in the defaults provided by BigBlueButton

Reply:
OK, I've removed this setting.